### PR TITLE
DRAFT: TS-reports.

### PR DESCRIPTION
**Pull request by ghost**
PR opened for discussion/feedback.

### CODE REVIEW

File: obip-ts-reports.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+__OBIP__: TS-Reports
```
Comment (michaelfolkson) [inline obip-ts-reports.md]: Header formatting from OBIP guidance (OBIP1) doesn't have each line starting in bold and has the complete header within HTML pre, /pre tags

File: obip-ts-reports.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+__Copyright__: MIT
+
+
+## Abstract
+A specification for the API a search provider should comply with to handle client reports for content.
+
+
+## Motivation
+Search providers have variying content policies and need a way to get reports of infringing content. This Proposal suggests an API for managing these reports.
```
Comment (michaelfolkson) [inline obip-ts-reports.md]: Typo - "varying"

File: obip-ts-reports.md (hunk 1 above)
Comment (ghost) [reply obip-ts-reports.md]: I like this better, but you're right. I'll change it and perhaps propose a change and see how people like it.

File: obip-ts-reports.md (hunk 2 above)
Comment (ghost) [reply obip-ts-reports.md]: Thanks, I will fix this. Speling isn't my stong suit.

**Pull request closed without merging**